### ISSUE TITLE
RotaryCraft Spring-powered Pump Not Placing Lava in Reservoir

### ISSUE THREAD

**Issue by Patman0021** [ReikaKalseki/Reika_Mods_Issues#1832]
When trying to empty my spring-powered pump of its lava, I'm forced to place it in world, then pick it up with a bucket and put it in a reservoir.  The pump being in "place" mode and right (or left, for that matter) clicking the reservoir does not transfer the lava.

**Comment by ReikaKalseki** [comment]
Does this only affect lava?

**Comment by Patman0021** [comment]
Hm.. Perhaps I reported too soon.  So, I can collect both water and lava in a spring-powered pump, and then by shift-right clicking on a reservoir, it will empty into it.  I cannot simply right click (while in place mode) on the reservoir and put one bucket, nor right click in drain mode on a reservoir and pull out one bucket.  
I found the shift-right click as I was trying everything I could think of.  
Close if this is as intended!  Thanks.
